src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.scss

```diff
@@ -1,50 +1,145 @@
-.media {
-  display: block;
-  min-height: 60px;
-  width: 100%;
-
-  .media-left {
-    border-top-left-radius: 2px;
-    border-top-right-radius: 0;
-    border-bottom-right-radius: 0;
-    border-bottom-left-radius: 2px;
-    display: block;
-    float: left;
-    height: 60px;
-    width: 60px;
-    text-align: center;
-    font-size: 40px;
-    line-height: 60px;
-    padding-right: 0;
-
-    .fa {
-      font-size: 45px;
-    }
-  }
-
-  .media-body {
-    padding: 5px 10px;
-    margin-left: 60px;
-
-    .media-heading {
-      display: block;
-      font-size: 20px;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
-
-      .underline {
-        border-bottom: 1px solid #e5e5e5;
-        margin-top: 0px;
-        margin-bottom: 18px;
-        padding: 0px;
-      }
-    }
-
-    .media-text {
-      display: block;
-      font-weight: bold;
-      font-size: 18px;
+$card-height: 8vw;
+$card-medium-height: 13vw;
+
+.card {
+  border: 0.5px solid #ddd;
+  border-radius: 3px;
+  background: #f5f5f5;
+  box-shadow: 0 1px 1px rgba(3, 3, 3, 0.175);
+  margin: 0 -10px 20px;
+  padding: 0 20px;
+  width: auto;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+
+  min-height: $card-height;
+}
+
+.card-title {
+  font-size: 1.2vw;
+  margin: 20px 0;
+  padding: 0;
+}
+
+.card-body {
+  font-size: 1.25vw;
```

Comment:
font-size seems a bit small when you use a xs screen.

Reply:
Ack.